CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.8.0] - 2020-11-24
+### Added
+- New config to support custom logger
```

Comment:
Is it worth noting the bugs you fixed here as well + calling out the change to how logger is configured to not use PxConfig due to circular dependency issues you identified?